tests/unit/facts/test_domain.py

```diff
@@ -40,3 +43,20 @@ def test_resolv_conf_no_domain(self, mock_fs_cat):
         facts_domain(self.dev, self.facts)
         self.assertEqual(self.facts['domain'], None)
         self.assertEqual(self.facts['fqdn'], 'test')
+
+    @patch('jnpr.junos.facts.domain.FS.cat')
```

Comment:
This test should patch jnpr.junos.utils.fs.FS.cat with a return value appropriate to complete the function.  The test assertion should be the proper facts are set.